src/serlio/utils/MayaUtilities.cpp

```diff
@@ -44,4 +44,14 @@ std::filesystem::path getWorkspaceRoot(MStatus& status) {
 		return {};
 	}
 }
+bool mStringArraysAreEqual(const MStringArray& lhs, const MStringArray& rhs) {
```

Comment:
consider implementing this as `operator==`, would be more versatile

Reply:
fixed.